src/main/kotlin/app/revanced/patches/reddit/layout/disablescreenshotpopup/fingerprints/DisableScreenshotPopupFingerprint.kt

```diff
@@ -0,0 +1,12 @@
+package app.revanced.patches.reddit.layout.disablescreenshotpopup.fingerprints
+
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+
+object DisableScreenshotPopupFingerprint : MethodFingerprint(
+    "V",
+    parameters = listOf("Landroidx/compose/runtime/e;", "I"),
```

Comment:
The first parameter type can change across versions as it looks minified. The class is named `e` currently. Consider removing this from the fingerprint.

Reply:
There are 2 methods with the same name in that class.

Reply:
> Is that parameter class (`e`) part of the apk, or part of android?
> 
> Could use the path but not the class name: `Landroidx/compose/runtime/`

It's part of the apk but framework related (jetpack compose) which i think is unlikely to change

Reply:
> Checking different versions of Reddit, the obfuscated classnames are different:
Reddit 2023.17.0:
public final void invoke(d dVar, int i12)

> Reddit 2023.23.0:
public final void invoke(e eVar2, int i18)

won't smali name will remain the same? since it's just variable type and not actual names.